webpack/assets/javascripts/react_app/components/HostStatuses/Status/index.js

```diff
@@ -0,0 +1,203 @@
+            'aria-label': 'Details',
+            'aria-labelledby': 'titleId toggle-button',
+            'aria-expanded': expanded,
+          }}
+        >
+          <Grid style={{ width: '100%' }} hasGutter>
+            <GridItem span={1} rowSpan={2} style={{ fontSize: '2.2em' }}>
+              <GlobalStatusIcon status={globalStatus} />
+            </GridItem>
+            <GridItem
+              span={unknownStatusesPresent ? 3 : 5}
+              style={{ fontSize: '1.5em' }}
+            >
+              {name}
+            </GridItem>
+            {unknownStatusesPresent && (
+              <GridItem span={2} rowSpan={2} style={{ textAlign: 'center' }}>
+                <div style={{ fontSize: '1.5em' }}>
+                  <GlobalStatusIcon />
+                </div>
+                <LinkOrLabel
+                  label={sprintf(__('Total: %s'), unknownTotalCount)}
+                />
+                <br />
+                <LinkOrLabel
+                  label={sprintf(__('Owned: %s'), unknownOwnedCount)}
+                />
+              </GridItem>
+            )}
+            <GridItem span={2} rowSpan={2} style={{ textAlign: 'center' }}>
+              <div style={{ fontSize: '1.5em' }}>
+                <GlobalStatusIcon status={0} />
+              </div>
+              <LinkOrLabel
+                path={okTotalPath}
+                label={sprintf(__('Total: %s'), okTotalCount)}
+              />
+              <br />
+              <LinkOrLabel
+                path={okOwnedPath}
+                label={sprintf(__('Owned: %s'), okOwnedCount)}
+              />
+            </GridItem>
+            <GridItem span={2} rowSpan={2} style={{ textAlign: 'center' }}>
+              <div style={{ fontSize: '1.5em' }}>
+                <GlobalStatusIcon status={1} />
+              </div>
+              <LinkOrLabel
+                path={warnTotalPath}
+                label={sprintf(__('Total: %s'), warnTotalCount)}
+              />
+              <br />
+              <LinkOrLabel
+                path={warnOwnedPath}
+                label={sprintf(__('Owned: %s'), warnOwnedCount)}
+              />
+            </GridItem>
+            <GridItem span={2} rowSpan={2} style={{ textAlign: 'center' }}>
+              <div style={{ fontSize: '1.5em' }}>
+                <GlobalStatusIcon status={2} />
+              </div>
+              <LinkOrLabel
+                path={errorTotalPath}
+                label={sprintf(__('Total: %s'), errorTotalCount)}
+              />
+              <br />
+              <LinkOrLabel
+                path={errorOwnedPath}
+                label={sprintf(__('Owned: %s'), errorOwnedCount)}
+              />
+            </GridItem>
+            <GridItem span={5}>{description}</GridItem>
+          </Grid>
+        </CardHeader>
+        <CardExpandableContent>
+          {details.length > 0 ? (
+            <Details data={details} />
+          ) : (
+            __('Nothing to show')
```

Comment:
how about render `emptyState` component?

Reply:
I would stick with just text, this component doesn't look good here